src/Dapr.AspNetCore/DaprBulkMessageEntry.cs

```diff
@@ -0,0 +1,67 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2021 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Dapr.AspNetCore
+{
+    /// <summary>
+    /// Represents a single event from a bulk of messages sent by the message bus.
+    /// </summary>
+    public class DaprBulkMessageEntry<TValue>
```

Comment:
So this is the actual response back? Or what we send out? I was confused with how `DaprBulkAppResponse` didn't have any events.

Reply:
`DaprBulkAppResponse ` is the response that we send back to dapr.